guides/common/modules/proc_disabling-cockpit-on-project.adoc

```diff
@@ -0,0 +1,30 @@
+[id="disabling-cockpit-on-project_{context}"]
+= Disabling {CockpitTitle} on {Project}
+
+Perform the following procedure if you want to disable {Cockpit} on {Project}.
+
+.Procedure
+. Disable {Cockpit} with the `{foreman-installer}` command:
```

Comment:
If I were to be pedantic, running the installer like this does not actually disable any related functionality. It just makes the installer not enforce the relevant configuration.

Reply:
Thank you. Updated the procedure :+1: